draft-irtf-cfrg-opaque.md

```diff
@@ -598,41 +609,42 @@ Steps:
 6. Output cleartext_credentials
 ~~~
 
-## Envelope Structure {#envelope-structure}
+## Key Recovery {#key-recovery}
```

Comment:
This seems like it belongs with the dependency section up top, where other parameters and constants are defined.

Reply:
What do you think?

I think that referencing `DeriveAuthKeyPair()` here makes sense for that the following functions.

Reply:
Or maybe we could add a line to show that this fills the requirement of compatibility with the AKE